info/v1/machine.go

```diff
@@ -26,6 +26,9 @@ type FsInfo struct {
 
 	// Total number of inodes available on the filesystem.
 	Inodes uint64 `json:"inodes"`
+
+	// HasInodes when true, indicates that Inodes info will be available.
+	HasInodes bool `json:"has_inodes"`
```

Comment:
hmm, can you model out what you would like to see to save some time?

The main intention in adding this was to keep this code from getting messy here:
https://github.com/google/cadvisor/blob/master/manager/manager.go#L671

I am not sure how to pull back the ContainerSpec cleanly from that context, but am happy to make the update with a pointer.